index.js

```diff
@@ -0,0 +1,28 @@
+require("dotenv").config();
+const aws = require("aws-sdk");
+
+(async function() {
+    try {
+        
+        aws.config.setPromisesDependency();
+
+        aws.config.update({
+            accessKeyId: process.env.ACCESS,
```

Comment:
I assume you've already tested (via logs or some other way) that these environment variables are coming through to the code?

Reply:
Yep. They are console logging just fine.

Reply:
They may need to be added to [environment.js](https://github.com/CruGlobal/cloudsearch-s3-handler/blob/master/serverless/environment.js)